.github/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# Changes from release 2019/05 to 2019/06
+x. Enhancement - Gatwick Airbridge Added - thanks to @nathaninnes (Nathan Innes)
```

Comment:
Just be as descriptive as possible in future, just to give people an idea of where they can see the changes for example. So "Added Gatwick (EGKK) Airbridge _to SMR_"

Reply:
Not a major issue though!